addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -1895,3 +1895,55 @@ QUnit.test("Notification settings: mute/unmute channel works correctly", async (
     await click("[title='Notification Settings']");
     await contains("span", { text: "Unmute Channel" });
 });
+
+QUnit.test("Notification settings: mute a channel should disable the web_push for notification", async (assert) => {
```

Comment:
It should be a python test as it is a python fix, maybe have a look at other python tests that reference `push_to_end_point`